ci_scripts/build_android.sh

```diff
@@ -17,7 +17,8 @@ cmake . -G Ninja -B "build-android-$ANDROID_ABI" \
 -DANDROID_NDK="$ANDROID_NDK" \
 -DCMAKE_TOOLCHAIN_FILE="$ANDROID_NDK/build/cmake/android.toolchain.cmake" \
 -DCMAKE_BUILD_TYPE=Release \
--DBASISU_SUPPORT_SSE=OFF
+-DBASISU_SUPPORT_SSE=OFF \
+-DISA_NONE=ON
```

Comment:
What is ISA - instruction set architecture? 

How about a single CMake option, say ASTCENC_ISA, that takes values of SSE2, SSE41, AVX2 and NEON. I think that will be easier for people to deal with. The acceptable values could be modified depending on the detection by cputypetest.

Reply:
> What is ISA - instruction set architecture?

Yes. Its what astcencoder chose to use. I did request a single option for `astcenc` as well but the rationale for this is to be able to build multiple static libraries at once for all SIMD options. Some clients are using astcenc like that.

We can indeed have an ASTCENC_ISA option for `toktx` which we can parse to write the right option in CMakeLists.txt. But this whole system should be changed in the next version of astcenc. So I would defer this to that time.

The `ISA_NONE` in particular is a build hack for android. I need to understand from you better why I am getting weird errors and then fix those separately.

Reply:
Agree to defer. I don't know much about the Android build. We may need to reach out to the person who set it up.